dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/PythonCommandExecutor.java

```diff
@@ -118,7 +118,7 @@ protected String commandInterpreter() {
         if (StringUtils.isEmpty(pythonHome)){
             return PYTHON;
         }
-        return pythonHome;
+        return pythonHome + "/bin/python";
```

Comment:
By searching `PYTHON_HOME` globally, also pay attention to modify the default value of `PYTHON_HOME